pkg/storage/ingest/fetcher.go

```diff
@@ -126,18 +126,6 @@ func (w fetchWant) expectedBytes() int {
 	return int(overFetchBytesFactor * float64(w.estimatedBytesPerRecord*int(w.endOffset-w.startOffset)))
```

Comment:
[nit] I would make `expectedBytes()` return a `int64`, just in case we have some OSS users running on 32bit systems (don't think so, but...).

Reply:
👍 https://github.com/grafana/mimir/pull/9919/commits/70fa210765ee7edfc97875266eb546240569b431